### ISSUE TITLE
Accessibility: The user is unable to toggle between payment methods using the keyboard

### ISSUE THREAD

**Issue by AndrewBarber** [bigcommerce/checkout-js#1726]
On checkout the user should be able to hit the tab key to navigate between different payment methods and then hit the enter key to select any of the highlighted payment methods.

However, if the user hits the tab key on payment methods this will skip past any payment methods displayed after the first one.

This will stop the user from selecting a desired payment method if they are limited to using a keyboard to navigate checkout.
